test/libgit2.jl

```diff
@@ -369,7 +369,7 @@ mktempdir() do dir
                 error("unexpected")
             catch e
                 @test typeof(e) == LibGit2.GitError
-                @test startswith(sprint(show,e),"GitError(Code:ENOTFOUND, Class:OS, Failed to resolve path")
+                @test startswith(sprint(show,e),"GitError(Code:ENOTFOUND, Class:OS, failed to resolve path")
```

Comment:
looks like this makes the windows test sad, since it uses libgit2 from the nightly binary instead of compiling from source

for the sake of distro packagers who might not be able to upgrade libgit2 versions right away, how about making this conditional on libgit2 version?